app/controllers/hiring_staff/vacancies/application_controller.rb

```diff
@@ -18,14 +20,18 @@ def session_vacancy_id
 
   def set_vacancy
     if params[:job_id]
-      @vacancy = current_school.vacancies.find(params[:job_id])
+      @vacancy = Vacancy.find(params[:job_id])
```

Comment:
`current_organisation.vacancies` ensures only vacancies for the current School or SchoolGroup can be found

Reply:
This was almost very bad security

Reply:
I'll change this when #1749 is merged to master.